open-company/open-company-web
# Logout issue

card: https://trello.com/c/CyxHgqGv

to test:
- deploy this on staging or beta
- remove the jwt
- login
- navigate the site and check your write permissions
- logout
- check that the jwt is gone from your cookies and check your write permissions (you should have readonly permissions now)
